### PR TITLE
test: faster init for sweat storage stress-test

### PR DESCRIPTION
**Pull request by jakmeier**
Use `prepare_accounts` (introduced in #9428)
to create the 1000 users required to start
the sweat based storage stress test.

Also, we don't have to send FT balance to those users,
or even add a key to the oracle account. Just creating the
account and sending some storage balance is good enough.

If running with 50 users and ramping up 1 user/sec,
this reduced the initialization time of the stress test
from 3m20s to 1m30s. Not perfect but better.

### CODE REVIEW

File: pytest/tests/loadtest/locust/locustfiles/sweat.py [hunk 1]
```diff
@@ -60,9 +60,19 @@ def record_single_batch(self):
     @tag("storage-stress-test")
     @task
     def record_batch_of_large_batches(self):
-        # ensure large enough state by creating more sweat users
+        # create more sweat users to allow for a decent record_batch size
         while len(self.sweat.registered_users) < 1000:
-            SweatUser(self.environment).on_start()
+            # creating 20 account in parallel, about 200 should fit in a chunk
```
Comment (akashin) [inline pytest/tests/loadtest/locust/locustfiles/sweat.py:65]: typo: account -> accounts

File: pytest/tests/loadtest/locust/common/ft.py [hunk 2]
```diff
@@ -60,6 +69,38 @@ def random_receivers(self, sender: str, num) -> typing.List[str]:
...
+        transactions. They are only used as targets of transactions and as a
+        side-effect they also increase the state size of the contract.
+        """
+        prefix_len = max_account_id_len - len(parent.key.account_id) - 1
+        assert prefix_len > 4, f"user key {parent.key.account_id} is too long"
+        chars = string.ascii_lowercase + string.digits
+        rng = random.Random()
+
+        def create_account():
+            prefix = ''.join(rng.choice(chars) for _ in range(prefix_len))
```
Comment (akashin) [inline pytest/tests/loadtest/locust/common/ft.py:89]: Same result, but a bit more idiomatic: `''.join(random.choices(chars, k=prefix_len))`

**Review by akashin — APPROVED** [review]
LGTM!

**Comment by jakmeier** [comment]
cc @Longarithm if you are still running locust sweat tests, you might want to pull this change in to speed up each test iteration

**Pull request merged**